app.js

```diff
@@ -1,81 +1,103 @@
-  await image.quality(100).writeAsync(outputFile);
-};
+  const addTextWatermarkToImage = async function(inputFile, outputFile, text) {
+    const image = await Jimp.read(inputFile);
+    const font = await Jimp.loadFont(Jimp.FONT_SANS_32_WHITE);
 
-const addImageWatermarkToImage = async function(inputFile, outputFile, watermarkFile) {
-  const image = await Jimp.read(inputFile);
-  const watermark = await Jimp.read(watermarkFile);
-  const x = image.getWidth() / 2 - watermark.getWidth() / 2;
-  const y = image.getHeight() / 2 - watermark.getHeight() / 2;
+    const textData = {
+      text,
+      alignmentX: Jimp.HORIZONTAL_ALIGN_CENTER,
+      alignmentY: Jimp.VERTICAL_ALIGN_MIDDLE,
+    };
 
-  image.composite(watermark, x, y, {
-    mode: Jimp.BLEND_SOURCE_OVER,
-    opacitySource: 0.5,
-  });
-  await image.quality(100).writeAsync(outputFile);
-};
+    image.print(font, 0, 0, textData, image.getWidth(), image.getHeight());
 
-const prepareOutputFilename = (filename) => {
-  const [ name, ext ] = filename.split('.');
-  return `${name}-with-watermark.${ext}`;
-};
+    await image.quality(100).writeAsync(outputFile);
+  };
 
-const startApp = async () => {
+  const addImageWatermarkToImage = async function(inputFile, outputFile, watermarkFile) {
+    const image = await Jimp.read(inputFile);
+    const watermark = await Jimp.read(watermarkFile);
+
+    const x = image.getWidth() / 2 - watermark.getWidth() / 2;
+    const y = image.getHeight() / 2 - watermark.getHeight() / 2;
+
+    image.composite(watermark, x, y, {
+      mode: Jimp.BLEND_SOURCE_OVER,
+      opacitySource: 0.5,
+    });
+    await image.quality(100).writeAsync(outputFile);
+  }
+
+  const prepareOutputFilename = filename => {
+    const [ name, ext ] = filename.split('.')
+    return `${name}` + `-with-watermark.` + `${ext}`;
+  };
 
   // Ask if user is ready
-  const answer = await inquirer.prompt([{
+
+  try {
+    const answer = await inquirer.prompt([{
       name: 'start',
-      message: 'Hi! Welcome to "Watermark manager". Copy your image files to `/img` folder. Then you\'ll be able to use them in the app. Are you ready?',
-      type: 'confirm'
+      message: 'Hi! Welcome to "Watermark manager". Copy your image files to `\img` folder. Then you\'ll be able to use them in the app. Are you ready?',
+      type: 'confirm',
     }]);
 
-  // if answer is no, just quit the app
-  if(!answer.start) process.exit();
-
-  // ask about input file and watermark type
-  const options = await inquirer.prompt([{
-    name: 'inputImage',
-    type: 'input',
-    message: 'What file do you want to mark?',
-    default: 'test.jpg',
-  }, {
-    name: 'watermarkType',
-    type: 'list',
-    choices: ['Text watermark', 'Image watermark'],
-  }]);
-
-  if(options.watermarkType === 'Text watermark') {
-    const text = await inquirer.prompt([{
-      name: 'value',
+    // if answer is no, just quit the app
+    if(!answer.start) process.exit();
```

Comment:
Dobrą praktyką jest używanie nawiasów klamrowych. Oczywiście technicznie dla warunku jednolinijkowego nie ma takiej potrzeby, ale czasami może to wygenerować błędy jeżeli przeoczymy, że zastosowaliśmy taki warunek i przez przypadek dodamy np. coś po ifie.